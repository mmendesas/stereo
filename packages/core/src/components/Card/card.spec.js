import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Card } from '.';

describe('Components | Card', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<Card>Card</Card>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
