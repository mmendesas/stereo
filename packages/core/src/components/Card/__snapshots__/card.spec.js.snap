// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components | Card renders correctly 1`] = `
.c0 {
  width: 300px;
  padding: 10px;
  margin: 15px;
  box-shadow: 0 5px 15px -5px rgba(0,0,0,1);
  border-radius: 5px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-transition: 0.3s;
  transition: 0.3s;
}

.c0:hover {
  box-shadow: 0 10px 30px -10px rgba(0,0,0,1);
}

<div
  className="c0"
>
  Card
</div>
`;
